cli/args/config_file.rs

```diff
@@ -922,8 +922,8 @@ pub fn get_ts_config_for_emit(
       "incremental": true,
       "jsx": "react",
       "importsNotUsedAsValues": "remove",
-      "inlineSourceMap": true,
-      "inlineSources": true,
+      "inlineSourceMap": false,
+      "inlineSources": false,
```

Comment:
I changed this to `false` because we don't use it anymore and it was appearing in a bitwise flag in the tsbuildinfo file (so might as well exclude it).